src/index.html

```diff
@@ -9,6 +9,167 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Human rights</h1>
+<h1>Universal Declaration of Human Rights</h1>
+<h2>Preamble</h2>
+<p>Whereas recognition of the inherent dignity and of the equal and inalienable
+rights of all members of the human family is the foundation of freedom, justice
+and peace in the world,<br>
+Whereas disregard and contempt for human rights have resulted in barbarous
+acts which have outraged the conscience of mankind, and the advent of a world
+in which human beings shall enjoy freedom of speech and belief and freedom
+from fear and want has been proclaimed as the highest aspiration of the common
```

Comment:
fix indentations;
when you write some text inside tags move it to the new line and add proper indentations so that the code looks clear